docs/index.rst

```diff
@@ -33,7 +33,7 @@ such as:
 on the :ref:`plugin_matrix` page.
 
 Quickstart
------------
+----------
```

Comment:
Thanks for catching this, I'm not sure why we're indenting by 1 to begin with, but this is a good find.